src/compas_rhino/install.py

```diff
@@ -123,6 +125,8 @@ def install(version=None, packages=None):
             with open(compas_bootstrapper, 'w') as f:
                 f.write('ENVIRONMENT_NAME = r"{}"\n'.format(environment_name))
                 f.write('PYTHON_DIRECTORY = r"{}"\n'.format(python_directory))
+                if conda_exe:
+                    f.write('CONDA_EXE = r"{}"\n'.format(conda_exe))
```

Comment:
why is this a conditional statement? if the exe is None, should that not also be reflected in the bootstrapper?

Reply:
mmm, just couldn't think of a case where bootstraper needs to declare 'conda not exist' rather than just skip providing the variable. 
otherwise we can change it to 
```python
if conda_exe:
    f.write('CONDA_EXE = r"{}"\n'.format(conda_exe))
else:
    f.write('CONDA_EXE = None\n'.)
```
But you see there still needs to be if condition, cuz it has to be assigned as real None value, not in quotes

Reply:
wait, I think assign empty string will do it